nslsii/utils.py

```diff
@@ -0,0 +1,40 @@
+import os
+import re
+import subprocess
+
+
+def run(cmd, path="", ignoreErrors=True, returnError=False, debug=False):
```

Comment:
I'd prefer to name the kwargs in the snake_case style, i.e.:
- `ignoreErrors` -> `ignore_errors`
- `returnError` -> `return_error`